tree/ntuple/v7/test/ntuple_minifile.cxx

```diff
@@ -82,7 +82,7 @@ TEST(MiniFile, Proper)
 
    auto rawFile = RRawFile::Create(fileGuard.GetPath());
    RMiniFileReader reader(rawFile.get());
-   auto ntuple = reader.GetNTuple("MyNTuple");
+   auto ntuple = reader.GetNTuple("MyNTuple").Get();
```

Comment:
Does 
```
RNTuple ntuple = reader.GetNTuple("MyNTuple");
```
If it does, isn't it 'better' than using the Get()?

Reply:
In this case it doesn't work because we can't coerce `RResult<RNTuple>` back to `RNTuple`. 
The `Get()` method is like "Give me the NTuple or immediately throw, I don't care about any errors (or I know there really is an NTuple there)"

Reply:
The `RResult<T>` type cannot automatically decay into `T`. We wanted to make this an explicit call because it involves verifying that we really have a valid object and not an exception in our hands.

Reply:
Fair enough. Thanks.